modules/prediction/network/net_layer.h

```diff
@@ -1,5 +1,5 @@
 /******************************************************************************
- * Copyright 2017 The Apollo Authors. All Rights Reserved.
+ * Copyright 2018 The Apollo Authors. All Rights Reserved.
```

Comment:
Please using the time the original file was uploaded.

Reply:
Thanks for clarifying--if this was a new file when we would have added the current year in the copyright notice.
Will revert this part of the changeset.